website/docs/docker/images.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Docker Images Hierarchy
+description: Docker Image Dependencies
+sidebar_position: 1
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import ThemedImage from '@theme/ThemedImage';
+
+:::info
+This page provides a detailed explanation of Docker 
+images and their hierarchy. For information on the process 
+of building these images, please refer to [this section](/docs/docker/deploy/index).
+:::
+
+
+The tools, scripts, dependencies, and configurations utilized by the 
+GATK-SV pipeline, written in WDL, are organized into separate Docker 
+containers. This modular approach ensures that each container 
+contains only the necessary tools for its specific task, 
+resulting in smaller image sizes. This design choice simplifies 
+the definition of Dockerfiles and facilitates easier maintenance. 
+Moreover, the smaller image sizes contribute to reduced disk 
+usage and lower workflow execution costs.
+
+
+The figure below illustrates the relationships between the GATK-SV Docker images.
```

Comment:
I think you need to explain the figure a bit. It wouldn't be clear to a naive reader what the arrows mean.

Reply:
Agree; I added a brief description.